gradle-baseline-java/src/main/resources/META-INF/gradle-plugins/com.palantir.baseline-checkstyle.properties

```diff
@@ -1 +0,0 @@
-implementation-class=com.palantir.baseline.plugins.BaselineCheckstyle
```

Comment:
These files are generated by the `gradlePlugin` config. It seems like we only declare about half of the plugins there. I've left the properties files for the ones that we don't declare there.

Ultimately we should declare them there, but I think that's outside the scope of this PR.

Reply:
Should probably not do this in the same PR.

Reply:
I have to at least remove these ones to stop using `include` in `gradle-baseline-java`.

Reply:
Ah, seems like the breaks from this are going to be pretty widespread then

Reply:
Oh wait, you mean the `'include'` change in baseline-only code

Reply:
Yes. I suppose I should probably pull that out into a separate PR.

Reply:
https://github.com/palantir/gradle-baseline/pull/3069